src/hotspot/share/interpreter/templateInterpreter.cpp

```diff
@@ -47,7 +47,9 @@ void TemplateInterpreter::initialize_stub() {
   // allocate interpreter
   int code_size = InterpreterCodeSize;
   NOT_PRODUCT(code_size *= 4;)  // debug uses extra interpreter code space
-  _code = new StubQueue(new InterpreterCodeletInterface, code_size, NULL,
+  int max_aligned_codelets = 280;
+  int max_aligned_bytes = max_aligned_codelets * CodeEntryAlignment * 2;
```

Comment:
Please explain in comment where these numbers (280, *2) are coming from and why you need additional size.